toolkit/themes/shared/design-system/design-tokens.json

```diff
@@ -55,6 +55,116 @@
       "value": "1px"
     }
   },
+  "box": {
+    "background": {
+      "color": {
+        "value": {
+          "light": "{color.white}",
+          "dark": "{color.gray.80}",
+          "prefersContrast": "{color.canvas}"
+        }
+      }
+    },
+    "shadow": {
+      "10": {
+        "value": "0 1px 4px {color.black.a10}"
+      }
+    }
+  },
+  "brand": {
+    "color": {
+      "accent": {
+        "@base": {
+          "value": {
+            "brand": {
+              "light": "{color.blue.50}",
+              "dark": "{color.cyan.50}"
+            }
+          }
+        },
+        "hover": {
+          "value": {
+            "brand": {
+              "light": "{color.blue.60}",
+              "dark": "{color.cyan.30}"
+            }
+          }
+        },
+        "active": {
+          "value": {
+            "brand": {
+              "light": "{color.blue.70}",
+              "dark": "{color.cyan.20}"
+            }
+          }
+        }
+      }
+    }
+  },
+  "button": {
+    "background": {
+      "color": {
+        "@base": {
+          "value": {
+            "prefersContrast": "ButtonFace",
+            "brand": {
+              "default": "color-mix(in srgb, currentColor 7%, transparent)"
+            },
+            "platform": {
+              "default": "var(--button-bgcolor)"
```

Comment:
We can't reference variables that aren't included in our JSON...but I'm pretty sure this will be moot once @mstriemer's button patches land: https://phabricator.services.mozilla.com/D188122#change-FJ7SeIB1CQgQ